src/commands/slow-mode.ts

```diff
@@ -0,0 +1,599 @@
+		.setFields(
+			{
+				name: 'User',
+				value: `<@${interaction.user.id}>`
+			},
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			},
+			{
+				name: 'Limit',
+				value: `1 message every ${limit} seconds`
+			}
+		).setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+
+	await sendEventLogMessage(interaction.guild!, channel.id, auditLogEmbed);
+
+	await interaction.followUp({ content: `Static slow mode enabled for <#${channel.id}>` });
+}
+
+async function disableSlowModeHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	await interaction.deferReply({ ephemeral: true });
+
+	const channel = interaction.options.getChannel('channel', false, [ChannelType.GuildText]) ?? interaction.channel;
+	if (!channel) {
+		throw new Error('No channel given');
+	}
+
+	if (channel.type !== ChannelType.GuildText) {
+		throw new Error('Slow mode only applies to text channels');
+	}
+
+	const slowMode = await SlowMode.findOne({
+		where: {
+			channel_id: channel.id
+		}
+	});
+
+	if ((!slowMode || !slowMode.enabled) && channel.rateLimitPerUser === 0) {
+		throw new Error(`Slow mode for <#${channel.id}> is already disabled`);
+	}
+
+	if (slowMode && slowMode.enabled === true) {
+		slowMode.enabled = false;
+		await slowMode.save();
+	}
+
+	await channel.setRateLimitPerUser(0);
+
+	const auditLogEmbed = new EmbedBuilder()
+		.setColor(0xC0C0C0)
+		.setTitle('Event Type: _Slow-Mode Disabled_')
+		.setDescription('――――――――――――――――――――――――――――――――――')
+		.setFields(
+			{
+				name: 'User',
+				value: `<@${interaction.user.id}>`
+			},
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			}
+		).setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+
+	await sendEventLogMessage(interaction.guild!, channel.id, auditLogEmbed);
+
+	await interaction.followUp({ content: `Slow mode disabled for <#${channel.id}>` });
+}
+
+async function slowModeStatsHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	await interaction.deferReply({ ephemeral: true });
+
+	const channel = interaction.options.getChannel('channel', false, [ChannelType.GuildText]) ?? interaction.channel;
+	if (!channel) {
+		throw new Error('No channel given');
+	}
+
+	if (channel.type !== ChannelType.GuildText) {
+		throw new Error('Slow mode only applies to text channels');
+	}
+
+	const slowMode = await SlowMode.findOne({
+		where: {
+			channel_id: channel.id
+		},
+		include: 'stages'
+	});
+
+	if (channel.rateLimitPerUser === 0) {
+		if (!slowMode || !slowMode.enabled) {
+			throw new Error(`No slow mode set for <#${channel.id}>`);
+		}
+	}
+
+	const embed = new EmbedBuilder()
+		.setTitle('Slow mode stats')
+		.setFields([
+			{
+				name: 'Channel',
+				value: `<#${channel.id}>`
+			},
+		])
+		.setFooter({
+			text: 'Pretendo Network',
+			iconURL: interaction.guild!.iconURL()!
+		});
+	
+	if (slowMode) {
+		if (slowMode.enabled) {
+			if (slowMode.users) {
```

Comment:
I think it would be useful to have a field that explicitly says whether the auto-slowmode is enabled or disabled.